frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/helpers.ts

```diff
@@ -0,0 +1,26 @@
+import { CD } from './constants';
+
+const compareCDS = (a, b) => {
+  const cdA = a.name.toUpperCase();
+  const cdB = b.name.toUpperCase();
+
+  let comparison = 0;
+  if (cdA > cdB) {
+    comparison = 1;
+  } else if (cdA < cdB) {
+    comparison = -1;
+  }
+  return comparison;
+};
+
+export const getAvailableCDName = (cds: CD[]) => {
+  let index = 1;
+  let name = `cd-drive-${index}`;
+  cds.sort(compareCDS).forEach((cd) => {
```

Comment:
I still think https://github.com/openshift/console/pull/3237#discussion_r352617792 is a better solution: 

Please do not modify the input array with sort as this is an unexpected behaviour for this function